pkg: change description and removal of com.miui.wmsvc and com.boundax.koreapreloadappinstaller

devCanario opened the pull request:
I think is recommended to remove because of the unsanitized SQL inputs, i didn't experienced any problems personaly as i said in the issue https://github.com/0x192/universal-android-debloater/issues/137.

Commit:
com.miui.wmsvc safe

Commit:
fix typo

AnonymousWP commented on resources/assets/uad_lists.json:
```diff
@@ -16328,11 +16328,11 @@
     "id": "com.miui.wmsvc",
     "list": "Oem",
-    "description": "WMService\nRuns at boot, has access to internet + GPS\nI quickly looked at the decompiled code and saw some unsanitized SQL inputs, which is BAD! (vulnerable to SQL injection)\nTries to get your android unique Google advertising ID from Google Play Services.\nFeeds and launches the spying/analytics app \"com.miui.hybrid\".\nDoesn't seem to do anything important, only tracking. ...
+    "description": "WMService\nRuns at boot, has access to internet + GPS\nI quickly looked at the decompiled code and saw some unsanitized SQL inputs, which is BAD! (vulnerable to SQL injection)\nTries to get your android unique Google advertising ID from Google Play Services.\nFeeds and launches the spying/analytics app \"com.miui.hybrid\".\nDoesn't seem to do anything important, only tracking. ...
```
```suggestion
    "description": "WMService\nRuns at boot, has access to internet + GPS\nI quickly looked at the decompiled code and saw some unsanitized SQL inputs, which is BAD! (vulnerable to SQL injection)\nTries to get your android unique Google advertising ID from Google Play Services.\nFeeds and launches the spying/analytics app \"com.miui.hybrid\".\nDoesn't seem to do anything important, only tracking.\nWARNING: Some people said removing this package causes bootloop, others said it doesn't. Safe to remove — no one seems to experience bootloops on MIUI 12 and higher.\nSee https://github.com/0x192/universal-android-debloater/issues/137 for more information.",
```

devCanario committed:
Update resources/assets/uad_lists.json

Co-authored-by: AnonymousWP <50231698+AnonymousWP@users.noreply.github.com>

Commit:
korea installer

Commit:
korea installer typo

AnonymousWP commented on resources/assets/uad_lists.json:
```diff
@@ -15920,6 +15920,15 @@
+  {
+    "id": "com.boundax.koreapreloadappinstaller",
+    "list": "Oem",
+    "description": "서비스 앱 설치 (Install the Services app), Didn`t found much info about this app, problaby safe to remove for all outside of korea \nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```
Can you please use English for the name of the app?
```suggestion
    "description": "서비스 앱 설치 (Install the Services app), Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```

devCanario committed:
Update resources/assets/uad_lists.json

Co-authored-by: AnonymousWP <50231698+AnonymousWP@users.noreply.github.com>

AnonymousWP commented on resources/assets/uad_lists.json:
```diff
@@ -15920,6 +15920,15 @@
+  {
+    "id": "com.boundax.koreapreloadappinstaller",
+    "list": "Oem",
+    "description": "서비스 앱 설치 (Install the Services app), Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```
Could you also translate the Korean part (and use another wording that's between the parentheses)?

devCanario replied:
is translated on the parenthesis the Korean works, do you mean that it should be in quotes?

AnonymousWP replied:
> is translated on the parenthesis the Korean works, do you mean that it should be in quotes?

Remove the Korean part and only keep the English string. I don't get how "Install the Services app" is the name of the app. The first sentence should always be the name of the app.

Commit:
Removed Korean characters

AnonymousWP commented on resources/assets/uad_lists.json:
```diff
@@ -15923,7 +15923,7 @@
     "id": "com.boundax.koreapreloadappinstaller",
     "list": "Oem",
-    "description": "서비스 앱 설치 (Install the Services app), Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
+    "description": "Install the Services app, Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```
I still think "Install the Service app" is not a correct name of an app, but more like a literal translation. Any alternatives for that?

devCanario replied:
Cant think of a better name for it the apk filename is skt-appinstaller.apk i can share the apk file if you want

AnonymousWP commented on resources/assets/uad_lists.json:
```diff
@@ -15920,6 +15920,15 @@
+  {
+    "id": "com.boundax.koreapreloadappinstaller",
+    "list": "Oem",
+    "description": "Install the Services app, Didn`t find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```
```suggestion
    "description": "App installer\nDidn't find much info about this app, probably safe to remove for everyone outside Korea\nhttps://beta.pithus.org/report/5c8127cd32cf61ec1382b05dca622a971242283a6f60f774574abee8c0774873",
```

AnonymousWP replied:
What about this?

devCanario replied:
i think that could work

devCanario committed:
Update resources/assets/uad_lists.json

Co-authored-by: AnonymousWP <50231698+AnonymousWP@users.noreply.github.com>

AnonymousWP committed:
pkg(miui): add missing packages

AnonymousWP committed:
pkg: change description of MIUI 14 packages

AnonymousWP committed:
pkg: add Redmi 8T packages

AnonymousWP committed:
pkg: add Xiaomi Pad 5 Pro packages

mmrabbani committed:
Update uad_lists.json

Add Xiaomi App vault

AnonymousWP committed:
pkg(`com.google.android.packageinstaller`): change description

mmrabbani committed:
Update resources/assets/uad_lists.json

Co-authored-by: AnonymousWP <50231698+AnonymousWP@users.noreply.github.com>

AnonymousWP committed:
docs(readme): fix typo and list

AnonymousWP committed:
pkg(`com.samsung.app.newtrim`): change description

AnonymousWP committed:
pkg(`com.samsung.android.hdmapp`): add missing list, description and change removal

AnonymousWP committed:
pkg: package documentation for Infinix

AnonymousWP committed:
pkg(`com.bbk.theme`): fix incorrect package name

LTeder committed:
Specify `iced` and `iced_native` versions in dependencies

`iced_native` was renamed to `iced_runtime` in `iced` version 0.10.0
Removed extraneous parameter in scrollable::StyleSheet::hovered in style.rs (added in 69235a9aa5c0fa3538af055761fabe1d69e720e4)
Bumping versions produces more errors

b02860de585071a2 committed:
Updated package description for com.samsung.vvm (https://github.com/0x192/universal-android-debloater/issues/876), added several new package descriptions (https://github.com/0x192/universal-android-debloater/issues/877).

Alliegaytor committed:
pkg: `com.android.cellbroadcastreceiver.module`

th4s committed:
Fix dependency proc-macro2

hacki904 committed:
Add files via upload

anuraggr committed:
Extended description for com.google.android.googlequicksearchbox

hacki904 committed:
Add files via upload

eltociear committed:
docs: update README.md

intructions -> instructions

devCanario committed:
Merge pull request #1 from devCanario/838-unlisted-miui-14-+-fixes-documentation

838 unlisted miui 14 + fixes documentation
